src/Serilog.Sinks.Couchbase/LoggerConfigurationCouchBaseExtensions.cs

```diff
@@ -38,22 +41,23 @@ public static class LoggerConfigurationCouchbaseExtensions
         /// <exception cref="ArgumentNullException">A required parameter is null.</exception>
         public static LoggerConfiguration Couchbase(
             this LoggerSinkConfiguration loggerConfiguration,
-            string[] couchbaseUriList, 
+            List<Uri> couchbaseUriList, 
             string bucketName, 
+            string bucketPassword = "",
```

Comment:
To reduce breaking changes where parameters are provided positionally in existing code, it might be better to put the new optional arg last? I'd also pass null as the default here, since the semantics "not provided" and "empty" are distinct that way.